tests/contracts/root_chain/test_piggyback_in_flight_exit.py

```diff
@@ -132,3 +133,56 @@ def test_piggyback_in_flight_exit_twice_should_fail(testlang):
     testlang.piggyback_in_flight_exit_input(spend_id, input_index, owner.key)
     with pytest.raises(TransactionFailed):
         testlang.piggyback_in_flight_exit_input(spend_id, input_index, owner.key)
+
+
+@pytest.mark.parametrize("num_outputs", [1, 2, 3, 4])
+def test_piggyback_in_flight_exit_output_with_preexisting_standard_exit_should_fail(testlang, num_outputs):
```

Comment:
I'm not getting into the details of these assertions, but there seem to be more `contract:` clauses in the docs that new tests (roughly). Pls, double/triple-check, whether all of the cases are covered here (ideally enumerate them in test docs/comments)

Reply:
Done